### PR TITLE
Firebase v6

### PR DESCRIPTION
**Pull request by guidoprinc**
## Summary

Added new configuration for Firebase Analytics
Deleted Google Analytics configuration
Added new configuration For Firebase Cloud Messaging
Added Firebase Performance

## Trello's
https://trello.com/c/H6bU2Mga
https://trello.com/c/NqrPSInJ
https://trello.com/c/WlYk8jo8

### CODE REVIEW

File: generators/app/templates/src/config/pushNotifications.js [hunk 1]
```diff
@@ -1,52 +1,75 @@
...
-try {
-  PushNotification = require('react-native-push-notification'); // eslint-disable-line global-require
-} catch (e) {} // eslint-disable-line no-empty, prettier/prettier
+import PushNotification from 'react-native-push-notification';
+import AsyncStorage from '@react-native-community/async-storage';
+import messaging from '@react-native-firebase/messaging';
 
-import { actionCreators as notificationActions } from '@redux/pushNotifications/actions'; // eslint-disable-line import/first
+let lastId = 0;
+const senderId = 'on-sender-id';
```
Comment (sfernandez11) [inline generators/app/templates/src/config/pushNotifications.js]: ⚠️ Change for a more generic `senderId`

File: generators/app/index.js [hunk 2]
```diff
@@ -48,9 +48,10 @@ class ReactNativeBootstrap extends Generator {
           'Drawer',
           'Crashlytics',
           'Push Notifications',
-          'Google Analytics',
+          'Firebase Analytics',
           'Redux Persist',
-          'Bitrise'
+          'Bitrise',
+          'Firebase Performance'
```
Comment (sfernandez11) [inline generators/app/index.js]: Change these features on the bootstrap [README](https://github.com/Wolox/wolmo-bootstrap-react-native/blob/8030db265b57de3753852e5f2333ac5763c74b22/README.md) too.

File: generators/app/templates/README.ejs [hunk 3]
```diff
@@ -16,9 +16,6 @@ We highly recommend using [nvm](https://github.com/creationix/nvm) instead of br
 create the `.env` file on the root.
 ```
 API_BASE_URL=https://XXXXXXXX
-<%_ if(features.googleanalytics) { _%>
-ANALYTICS_TRACKING_ID=UA-XXXXX-Y
-<%_ } _%>
```
Comment (sfernandez11) [inline generators/app/templates/README.ejs]: Replace with `NOTIFICATIONS_SENDER_ID`:
```
<%_ if(features.pushnotifications) { _%>
NOTIFICATIONS_SENDER_ID=XXXXXXXX
<%_ } _%>
```

File: generators/app/templates/src/app/index.ejs [hunk 4]
```diff
@@ -20,13 +13,6 @@ const App = () => {
   
   useEffect(() => SplashScreen.hide(), []);
   useEffect(() => {
-    <%_ if(features.pushnotifications) { _%>
-    configPushNotifications(dispatch);
-    alertIfHuaweiDevice();
-    <%_ } _%>
```
Comment (sfernandez11) [inline generators/app/templates/src/app/index.ejs]: Why you delete this config? I think It's ok to configure Push Notifications right here:
```
  <%_ if(features.pushnotifications) { _%>
    configPushNotifications();
  <%_ } _%>
```

File: generators/app/index.js (hunk 2 above)
Comment (guidoprinc) [reply generators/app/index.js]: 👍

**Pull request merged**